modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/ResultsBar.js

```diff
@@ -10,6 +10,39 @@ import {ManagementToolbar} from 'frontend-js-components-web';
 import {navigate, sub} from 'frontend-js-web';
 import React, {useEffect, useRef} from 'react';
 
+function getResultText(searchValue, itemTotal, filterTotal) {
+	if (searchValue) {
+		if (itemTotal === 1) {
+			if (Liferay.FeatureFlags['LPS-198573'] && filterTotal) {
+				return Liferay.Language.get('x-result-for-x-with-filters');
+			}
+
+			return Liferay.Language.get('x-result-for-x');
+		}
+		else {
+			if (Liferay.FeatureFlags['LPS-198573'] && filterTotal) {
+				return Liferay.Language.get('x-results-for-x-with-filters');
+			}
+
+			return Liferay.Language.get('x-results-for-x');
+		}
```

Comment:
I am not sure if this is worth tuning to language, singular and plural. In English, there is for example `21 result`, which has singular form. Some other languages, like Slovene, have [different form for two values](https://en.wikipedia.org/wiki/Dual_(grammatical_number)). I'd just keep the same simplification as previous locale, just one value for plural (`x-results-for-x-with-filters`)